packages/dnb-eufemia/src/extensions/forms/Field/PostalCodeAndCity/PostalCodeAndCity.tsx

```diff
@@ -131,6 +131,7 @@ function PostalCodeAndCity(props: Props) {
         inputClassName="dnb-forms-field-postal-code-and-city__postal-code-input"
         inputMode="numeric"
         autoComplete="postal-code"
+        data-country-code={country}
```

Comment:
```suggestion
        data-country-code={countryCode}
```

Should be changed after https://github.com/dnbexperience/eufemia/pull/4572 is merged